lib/stringify.h

```diff
@@ -123,6 +123,21 @@ cstring toString(const void *value);
 
 char DigitToChar(int digit);
 
+/// Convert a string to lower case. This is not safe for non-ASCII strings.
+/// Performs the lowering operation in place.
+void lowerinPlace(std::string &s);
+
+/// Convert a string to lower case. This is not safe for non-ASCII strings.
```

Comment:
It is (probably) "safe" in a sense it will not crash. It is just not what might be expected (e.g. locale-aware UTF-8 lowercase).